packages/components/src/internal/components/project/ProjectLookAndFeelForm.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { FC, memo, useCallback, useState } from 'react';
+import { Button, Col, FormControl, Panel, Row } from 'react-bootstrap';
+
+import { Container } from "@labkey/api";
+
+import { DATE_FORMATS_TOPIC, HelpLink, JavaDocsLink } from '../../util/helpLinks';
+import { LabelHelpTip } from '../base/LabelHelpTip';
+import { getDateTimeFormat } from '../../util/Date';
+import { Alert } from '../base/Alert';
+import { FolderAPIWrapper, ProjectLAFOptions } from '../container/FolderAPIWrapper';
+import { resolveErrorMessage } from '../../util/messaging';
+import { incrementClientSideMetricCount } from '../../actions';
+import { LOOK_AND_FEEL_METRIC } from '../productnavigation/constants';
+
+interface Props {
+    api?: FolderAPIWrapper;
+    onSuccess?: (reload?: boolean) => void;
+    container?: Container;
+}
+
+const PROJECT_DATE_FORMAT_HELP = (
+    <LabelHelpTip title="Date-time format">
+        <p>
+            To control how a date or time value is displayed, provide a string format compatible with the Java{' '}
+            <JavaDocsLink urlSuffix="java/text/SimpleDateFormat.html">SimpleDateFormat</JavaDocsLink> class.
+        </p>
+        <p>Examples for Jan 20, 2023 01:45 PM:</p>
+        <table className="table-bordered margin-bottom help-table">
+            <thead>
+                <tr>
+                    <th>Format String</th>
+                    <th>Display Result</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>yyyy-MM-dd HH:mm</td>
+                    <td>2023-01-20 13:45</td>
+                </tr>
+                <tr>
+                    <td>MM/dd/yyyy hh:mm aa</td>
+                    <td>01/20/2023 01:45 PM</td>
+                </tr>
+                <tr>
+                    <td>dd-MM-yy</td>
+                    <td>20-01-23</td>
+                </tr>
+                <tr>
+                    <td>dd-MMM-yyyy</td>
+                    <td>20-Jan-2023</td>
+                </tr>
+            </tbody>
+        </table>
+
+        <p>
+            Learn more about using <HelpLink topic={DATE_FORMATS_TOPIC}>Date and Time formats</HelpLink> in LabKey.
+        </p>
+    </LabelHelpTip>
+);
+
+export const ProjectLookAndFeelFrom: FC<Props> = memo(props => {
+    const { api, onSuccess, container } = props;
+    const [dateTimeFormat, setDateTimeFormat] = useState<string>(getDateTimeFormat(container));
```

Comment:
Consider reducing the number of calls to `getDateTimeFormat`

```tsx
const [dateTimeFormat, setDateTimeFormat] = useState<string>(() => getDateTimeFormat(container));
```

Reply:
cool, I didn't know that